.github/workflows/builds.yml

```diff
@@ -83,7 +83,13 @@ jobs:
       run:
         shell: bash
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          apt install -y zstd
+          curl --retry 5 --retry-delay 10 --output deps.tar.zst https://acts.web.cern.ch/ACTS/ci/ubuntu-24.04/deps.v6.tar.zst
+          tar -xf deps.tar.zst -C /usr/local --strip-components=1
+          rm deps.tar.zst
```

Comment:
@paulgessinger, I decided to download the tar file like this. I didn't quite understand why Acts is doing this in a much more complicated way... :thinking: